### PR TITLE
Upgrade SDK

### PR DESCRIPTION
**Pull request by andrewblaney**

### CODE REVIEW

File: pages/api/contact-form.ts [hunk 1]
```diff
@@ -47,10 +47,10 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse<
     return res.status(500).json({ error: upsertCustomerRes.error.message });
   }
 
-  console.log(`Customer upserted ${upsertCustomerRes.data.id}`);
+  console.log(`Customer upserted ${upsertCustomerRes.data.customer.id}`);
 
   const upsertTimelineEntryRes = await client.upsertCustomTimelineEntry({
-    customerId: upsertCustomerRes.data.id,
+    customerId: upsertCustomerRes.data?.customer.id,
```
Comment (mattvagni) [inline pages/api/contact-form.ts]: Why is the `?` needed?

File: package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@radix-ui/react-popover": "^1.0.6",
-    "@team-plain/typescript-sdk": "^1.1.0",
+    "@team-plain/typescript-sdk": "^2.4.0",
```
Comment (jesushernandez) [inline package.json]: Let's fix this to 2.4.0

File: pages/api/contact-form.ts (hunk 1 above)
Comment (andrewblaney) [reply pages/api/contact-form.ts]: its not!

File: package.json (hunk 2 above)
Comment (andrewblaney) [reply package.json]: done 😄

**Pull request merged**